### ISSUE TITLE
Meta description case sensitivity

### ISSUE THREAD

**Issue by nicogies** [toptal/webdevchecklist.com#90]
```html
<meta name="Description" ...>
```
is not detected by the Chrome Extension.

According to W3C https://dev.w3.org/html5/spec-preview/the-meta-element.html#standard-metadata-names meta names are case-insensitive

Maybe meta description could be case insensitive ?
Thanks
